google-cloud-spanner/src/main/java/com/google/cloud/spanner/Statement.java

```diff
@@ -246,4 +248,56 @@ StringBuilder toString(StringBuilder b) {
     }
     return b;
   }
+
+  public static final class StatementFactory {
+    private final Dialect dialect;
+
+    StatementFactory(Dialect dialect) {
+      this.dialect = dialect;
+    }
+
+    public Statement of(String sql) {
+      return Statement.of(sql);
+    }
+
+    /**
+     * @param sql SQL statement with unnamed parameter denoted as ?
+     * @param values list of values which needs to replace ? in the sql
+     * @return Statement object
```

Comment:
Either remove the `@return` tag or add a description of what is returned, e.g.:
```suggestion
     * @return Statement a statement that can be executed on Spanner 
```

Reply:
Done